cmd/mavenInstall.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/maven"
+	"github.com/SAP/jenkins-library/pkg/piperutils"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+)
+
+func mavenInstall(config mavenInstallOptions, telemetryData *telemetry.CustomData) {
+	// for command execution use Command
+	c := command.Command{}
+	// reroute command output to logging framework
+	c.Stdout(log.Entry().Writer())
+	c.Stderr(log.Entry().Writer())
+
+	// for http calls import  piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	// and use a  &piperhttp.Client{} in a custom system
+	// Example: step checkmarxExecuteScan.go
+
+	// error situations should stop execution through log.Entry().Fatal() call which leads to an os.Exit(1) in the end
+	err := runMavenInstall(&config, telemetryData, &c)
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+}
+
+func runMavenInstall(config *mavenInstallOptions, telemetryData *telemetry.CustomData, command execRunner) error {
+	var flags = []string{"-update-snapshots", "--batch-mode"}
+
+	exists, _ := piperutils.FileExists("integration-tests/pom.xml")
```

Comment:
We have that already implemented in `pkg/maven`

Reply:
But here I just want to exclude the integration tests